### PR TITLE
Add namespace flag to oc create

### PR DESCRIPTION
**Pull request by wshearn**

### CODE REVIEW

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_serviceaccounts/tasks/main.yml [hunk 1]
```diff
@@ -9,7 +9,8 @@
 - name: create the service account
   shell: >
        echo {{ lookup('template', '../templates/serviceaccount.j2')
-               | from_yaml | to_json | quote }} | {{ openshift.common.client_binary }}  create -f -
+               | from_yaml | to_json | quote }}
+               | {{ openshift.common.client_binary }} -n {{ openshift_serviceaccounts_namespace }} create -f -
```
Comment (abutcher) [inline roles/openshift_serviceaccounts/tasks/main.yml]: It doesn't like the pipe there but this works for me

```
- name: create the service account
  shell: >
    echo {{ lookup('template', '../templates/serviceaccount.j2')
    | from_yaml | to_json | quote }} | {{ openshift.common.client_binary }}
    -n {{ openshift_serviceaccounts_namespace }} create -f -
  when: item.1.rc != 0
  with_together:
  - openshift_serviceaccounts_names
  - account_test.results
```

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_serviceaccounts/tasks/main.yml [hunk 2]
```diff
@@ -9,7 +9,8 @@
 - name: create the service account
   shell: >
        echo {{ lookup('template', '../templates/serviceaccount.j2')
-               | from_yaml | to_json | quote }} | {{ openshift.common.client_binary }}  create -f -
+               | from_yaml | to_json | quote }} | {{ openshift.common.client_binary }}
+               -n {{ openshift_serviceaccounts_namespace }} create -f -
```
Comment (abutcher) [inline roles/openshift_serviceaccounts/tasks/main.yml]: The indentation is the problem. We can bring each everything under `shell: >` to a two space indent.

**Comment by wshearn** [comment]
aos-ci-test

**Comment by abutcher** [comment]
:+1:

**Comment by abutcher** [comment]
@brenton

**Pull request merged**